lib/workers/global/config/parse/file.ts

```diff
@@ -71,6 +71,14 @@ export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
       logger.debug('No config file found on disk - skipping');
     }
   }
+
+  if (
+    env.RENOVATE_CONFIG_FILE &&
```

Comment:
I don't mind too much, but I was surprised to see that this env variable is a requirement for deleting

Reply:
for safety we didnt want to remove the default `config.js`, we can tho

Reply:
You probably want to mention it in a comment then